data/json/items/ammo/3006.json

```diff
@@ -17,7 +17,7 @@
     "ammo_type": "3006",
     "casing": "3006_casing",
     "range": 61,
-    "damage": { "damage_type": "bullet", "amount": 62, "armor_penetration": 10 },
+    "damage": { "damage_type": "bullet", "amount": 49, "armor_penetration": 10 },
```

Comment:
Changes to 30-06 should probably be duplicated on 12.3ln, which supposedly has identical ballistic performance.